build-release.sh

```diff
@@ -10,7 +10,8 @@ export OUT_DIR="${1-dist}"
 export VERSION=${2:-$(git describe --tags --always --dirty)}
 
 # To overwrite the version details, pass something as the third arg. Empty string disables it.
-export VERSION_DETAILS=${3-"$(date -u +"%FT%T%z")/$(git describe --tags --always --long --dirty)"}
+export COMMIT_DATE=$(git log -1 --format=%ct $VERSION | { read d ; date -u +"%FT%T%z" -d @"$d" ; })
```

Comment:
```suggestion
export COMMIT_DATE=$(git log -1 --format=%ct HEAD | { read d ; date -u +"%FT%T%z" -d @"$d" ; })
```

I feel like this is what you meant ?

You also can just drop `HEAD` I guess.